SYCL/ESIMD/regression/Inputs/dgetrf.hpp

```diff
@@ -47,7 +53,7 @@ using namespace sycl;
 using namespace std;
 using namespace sycl::ext::intel::esimd;
 
-ESIMD_PRIVATE ESIMD_REGISTER(192) simd<double, 3 * 32 * 4> GRF;
+ESIMD_PRIVATE ESIMD_REGISTER(256) simd<fptype, 3 * 32 * 4> GRF;
```

Comment:
@intel/dpcpp-esimd-reviewers , is this change (192->256) ok?

Reply:
Revert the change back to `192`, this is an internal requirement, should not merge to public repo.